pipeline/data_merging/brca_pseudonym_generator.py

```diff
@@ -231,16 +238,24 @@ def main(input, output, log_path, config_file, resources, processes):
     df[GENOMIC_HGVS_HG38_COL] = df[TMP_HGVS_HG38].apply(str)
 
     logging.info("Compute hg37 representation of internal representation")
-    var_objs_hg37 = convert_to_hg37(df[VAR_OBJ_FIELD], resources)
+    var_objs_hg37, var_objs_hg37_failed = convert_to_hg37(df[VAR_OBJ_FIELD], resources)
 
     logging.info("Compute hg37 normalized representation of internal")
     # normalizing again for the hg37 representation. An alternative would be to convert the normalized hg38 representation to hg37.
     # If we use crossmap, we would need a way to convert the VCF like representation back to an hgvs object, which we currently
     # are unable to do properly. That is, we can use VCFVariant.to_hgvs_obj, however, structural variants will be converted
     # to delins, losing information if a variant was e.g. a del, ins, or dup.
 
-    df[TMP_HGVS_HG37] = pd.Series(
-        [v.to_hgvs_obj(hgvs_proc.contig_maps[HgvsWrapper.GRCh37_Assem]) for v in var_objs_hg37])
+    # set None values for any hg37 coordinates that cannot be derived and add to lists in proper order
+    tmp_hgvs_hg37_values = [v.to_hgvs_obj(hgvs_proc.contig_maps[HgvsWrapper.GRCh37_Assem]) for v in var_objs_hg37]
+    for v in var_objs_hg37_failed:
+        hgvs_obj = v.to_hgvs_obj(hgvs_proc.contig_maps[HgvsWrapper.GRCh38_Assem])
+        row_number = df[df[GENOMIC_HGVS_HG38_COL] == str(hgvs_obj)].index[0]
```

Comment:
instead recomputing the hgvs_obj you could compare to the `VAR_OBJ_FIELD`:
```
row_number = df[df[VAR_OBJ_FIELD] == v].index[0]
```
This is simpler and probably faster.

Reply:
This was my first approach too :) In theory it would work, but there's an issue with formatting (notice the '' around the chr value):
```
(Pdb) df[VAR_OBJ_FIELD].iloc[9]
VCFVariant(chr=11, pos=108312440, ref='G', alt='A')
(Pdb) v
VCFVariant(chr='11', pos=108312440, ref='G', alt='A')
(Pdb) df[df[VAR_OBJ_FIELD] == v].index[0]
*** IndexError: index 0 is out of bounds for axis 0 with size 0
```

For what it's worth, there's only one problematic variant of >80,000 at the moment. I think we can take the performance hit of recomputing hgvs_obj 😁